src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -132,18 +148,20 @@ function WorkspaceWorkflowsPage({policy, betas, route, reimbursementAccount, ses
                         shouldShowRightIcon
                         wrapperStyle={containerStyle}
                         hoverAndPressStyle={[styles.mr0, styles.br2]}
+                        brickRoadIndicator={hasApprovalError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
                     />
                 ),
-                isActive: policy?.isAutoApprovalEnabled ?? false,
+                isActive: (policy?.isAutoApprovalEnabled && !hasApprovalError) ?? false,
                 pendingAction: policy?.pendingFields?.approvalMode,
+                errors: ErrorUtils.getLatestErrorField(policy ?? {}, CONST.POLICY.COLLECTION_FIELDS.APPROVAL_MODE),
+                onCloseError: () => Policy.clearPolicyErrorField(policy?.id ?? '', CONST.POLICY.COLLECTION_FIELDS.APPROVAL_MODE),
             },
             {
                 icon: Illustrations.WalletAlt,
                 title: translate('workflowsPage.makeOrTrackPaymentsTitle'),
                 subtitle: translate('workflowsPage.makeOrTrackPaymentsDescription'),
                 onToggle: () => {
-                    const isActive = policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES;
-                    const newReimbursementChoice = isActive ? CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_MANUAL : CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES;
+                    const newReimbursementChoice = !hasVBA ? CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_MANUAL : CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES;
```

Comment:
This will never toggle on from a clean state with no VBA, as we will set to `REIMBURSEMENT_MANUAL` initially, whereas we consider only `REIMBURSEMENT_YES` as the active state here:

https://github.com/Expensify/App/blob/7c95b21b5ead9f1981b7226ca55c1805514d9661/src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx#L212

It also won't toggle off as we don't set `REIMBURSEMENT_NO`.